### ISSUE TITLE
{Freeway-Bug}[Adaptive Cards - Activity Update] - Ensures role attribute has an appropriate value for the element.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8350]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS: Version 22H2 (OS Build 25300.1000)
URL: https://adaptivecards.io/samples/ActivityUpdate.html
Browser: Edge Dev Version 111.0.1661.36 (Official build) dev (64-bit)
Screen Reader: Narrator

**Repro Steps:**

1. Open the above URL in edge dev browser.
2. Turn on the fast pass of accessibility insights for web.
3. Observe the issue.

**Actual Result:**
Role attribute does not have an appropriate value for the element.

**Expected Result:**
Ensures role attribute should have an appropriate value for the element.

**Snippet:**
< open anchor tag >role="listitem" class="selectedHolder" aria-current="true" aria-posinset="1" aria-setsize="29" href="/samples/ActivityUpdate.html">Activity update <Close anchor tag> 

**How to fix:** 
Fix any of the following:
  ARIA role listitem is not allowed for given element

**User Impact:**
Screen reader user will get impacted if incorrect role is defined due to which user will not be able to understand the purpose of the content easily.

**WCAG Reference link:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html

“Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-vaibgu%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400980259288%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=TrcBaCyAPKg0WAQfdSZJP2i9pghMEDg59i2NeUUs5EQ%3D&reserved=0) “

### Screenshots

![image](https://user-images.githubusercontent.com/95913748/223420582-f6abe06d-01c3-43df-ada3-315b4286802c.png)

![WCAG4 1 2-Ensures role attribute has an appropriate value for the element](https://user-images.githubusercontent.com/95913748/224946105-b83f94b8-ac22-42ab-9066-4d8a667d9fe5.png)

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
Reactivating this issue as this issue is still repro on the environment below.

**Test Environment:**
OS: Version 23H2 (OS Build 25381.1)
URL: https://adaptivecards.io/samples/ActivityUpdate.html
Browser: Edge Dev Version 115.0.1880.3 (Official build) dev (64-bit)
Screen Reader: Narrator

**Snippet:**

![image](https://github.com/microsoft/AdaptiveCards/assets/95913748/1008dd05-acfe-407f-a60d-cab6800f7284)

**Comment by anna-dingler** [comment]
Hi @vagpt,

We just made a release of our website, and this change should now be available. Thanks!

**Comment by vagpt** [comment]
This issue is not repro on the environment below, hence closing this issue.

Test Environment:
URL: https://adaptivecards.io/designer/
OS Version: 23H2(OS Build 25931.1000)
Browser Version: 118.0.2060.1
